src/pages/VideoPlayer/RelatedVideos.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Item } from 'semantic-ui-react';
+import { useHistory } from 'react-router-dom';
+
+function RelatedVideos({ related }) {
+    const history = useHistory();
+    const renderGroup = () => {
+        const items = [];
+        related.forEach((video, i) => {
+            items.push({
+                childKey: i,
+                image: video?.snippet?.thumbnails?.high?.url,
+                header: video?.snippet?.title,
+                as: 'a',
+                description: '',
+                title: 'card',
+                meta: video?.snippet?.channelTitle,
+                onClick: () => history.push('/video', {video: video})
+              })
+        })
+        return items;
+    };
+
+    return (
+        <Item.Group title={'relatedList'} divided items={renderGroup()} />
+    );
```

Comment:
Here `renderGroup` is being defined as a function which is called immediately after. Inside an empty array is defined and a `forEach` is used to push new items to the array and lastly the array is returned. We can make this more `js-friendly` by changing it to:

```suggestion
    const items = related.map((video, i) => ({
      childKey: i,
      image: video?.snippet?.thumbnails?.high?.url,
      header: video?.snippet?.title,
      as: 'a',
      description: '',
      title: 'card',
      meta: video?.snippet?.channelTitle,
      onClick: () => history.push('/video', { video: video })
    }));

    return (
        <Item.Group title={'relatedList'} divided items={items} />
    );
```